templates/dex.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: dex
+  namespace: kubeaddons
+spec:
+  kubernetes:
+    minSupportedVersion: v1.14.0
+  chartReference:
+    chart: stable/dex
+    version: 1.4.0
+    values: |
+      ---
+      image: mhrabovcin/dex
```

Comment:
shouldn't we move this docker image under mesosphere ?

Reply:
I am going to propose changes to upstream `dex` repository and ideally, we'll be running off official container. If we won't get changes upstream in the time I am planning to push custom version to our `mesosphere` namespace.

Reply:
As this is high prio and it appears a suitable answer was provided for the concern, I'm going to mark this as resolved but please reopen and flag request changes if this is not the case.